File: client/core/api/models/template-plane.ts
import { Plane } from './plane';
import { Template } from './template';
import { TemplatePerson } from './template-person';

export interface TemplatePlane {
  id: number;
  planeId: number;
  templateId: number;

  plane: Plane;
  template: Template;

  templatePeople: TemplatePerson[];
}

File: server/AppSupport.Data/Entities/TemplatePlane.cs
using System.Collections.Generic;

namespace AppSupport.Data.Entities
{
    public class TemplatePlane
    {
        public int Id { get; set; }
        public int PlaneId { get; set; }
        public int TemplateId { get; set; }

        public Plane Plane { get; set; }
        public Template Template { get; set; }

        public virtual ICollection<TemplatePerson> TemplatePeople { get; set; }
    }
}